### PR TITLE
Roxygen update

### PR DESCRIPTION
**Pull request by arrismo**
@rudeboybert I have converted bayes_cluster.R to roxygen2 format
I created a new folder titled "old_man" to store the old .Rd files

### CODE REVIEW

File: R/bayes_cluster.R [hunk 1]
```diff
@@ -1,48 +1,129 @@
...
+#'
+#' @return 
+#' List containing
+#' return(list(
+#' prior.map=prior.map, 
+#' post.map=post.map, 
+#' pk.y=pk.y))
+#' \item{prior.map}{A list containing, for each area: 1) \code{high.area} the prior probability of cluster membership, 2) \code{low.area} anti-cluster membership, and 3) \code{RR.est.area} smoothed prior estimates of relative risk}
+#' \item{post.map}{A list containing, for each area: 1) \code{high.area} the posterior probability of cluster membership, 2) \code{low.area} anti-cluster membership, and 3) \code{RR.est.area} smoothed posterior estimates of the relative risk}
+#' \item{pk.y}{posterior probability of k clusters/anti-clusters given y for k=0,...,J}
```
Comment (rudeboybert) [inline R/bayes_cluster.R:27]: This is a good start! Time to level up:

1. Be sure to inspect the `?bayes_cluster` outputs as you go. but it seems like you're doing this.
1. While the package "builds" in that you can install it, "build" is a very low-threshold of quality to facilitate rapid iteration and development. To ensure a package passes CRAN (R package app store) standards, you need to run a package "check": https://r-pkgs.org/whole-game.html#check. You can do this in RStudio > Build Tab > Check
2. Also be sure to copy over the example code. See this [example](https://github.com/moderndive/moderndive/blob/master/R/ggplot_parallel_slopes.R#L26) of loading example code.

Comment (rudeboybert) [reply R/bayes_cluster.R:27]: Try to fix all the `check()` errors you can; hmu if you get stuck.

Once you're done the next round of edits, commit them, then run `roxygen2md::roxygen2md()` and then look at the git diff of `bayes_cluster.R`. It converts a lot of the old formatting of roxygen code to use markdown formatting, which is way cleaner. After you've looked at the differences, commit those changes, and push all

File: DESCRIPTION [hunk 2]
```diff
@@ -39,3 +39,4 @@ Suggests:
 Config/testthat/edition: 3
 VignetteBuilder: knitr
 Encoding: UTF-8
+Roxygen: list(markdown = TRUE)
```
Comment (rudeboybert) [inline DESCRIPTION:42]: This is where I enabled using markdown for roxygen code. I did this using `usethis::use_roxygen_md()`

File: R/bayes_cluster.R (hunk 1 above)
Comment (arrismo) [reply R/bayes_cluster.R:29]: HELP I am stuck on step 2

Comment (rudeboybert) [reply R/bayes_cluster.R:29]: I removed the blank and dashed lines in the `@example` code in 104ee99. that did it

**Comment by arrismo** [comment]
@rudeboybert  
ISSUE:
```
Running examples in ‘SpatialEpi-Ex.R’ failed
  The error most likely occurred in:

```

File: R/bayes_cluster.R [hunk 3]
```diff
@@ -37,47 +37,32 @@
 #' @export
 #' 
 #' @examples 
+#' ## Note for the NYleukemia example, 4 census tracts were completely surrounded 
+#' ## by another unique census tract; when applying the Bayesian cluster detection 
+#' ## model in \code{\link{bayes_cluster}}, we merge them with the surrounding 
+#' ## census tracts yielding \code{n=277} areas.
 #' 
-#' bayes_cluster <-
```
Comment (rudeboybert) [inline R/bayes_cluster.R]: Hey @arris the problem was lines 41-47. These are not part of the example code seen in the [original help file](https://github.com/rudeboybert/SpatialEpi/blob/master/man/bayes_cluster.Rd#L43), so removing them fixed the issue

**Pull request merged**